packages/client/ui/Select/SelectContent.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
+import * as RadixSelect from '@radix-ui/react-select'
+import {twMerge} from 'tailwind-merge'
+
+export const SelectContent = React.forwardRef<
+  HTMLDivElement,
+  React.ComponentPropsWithoutRef<typeof RadixSelect.Content>
+>(({className, children, position = 'popper', ...props}, ref) => (
+  <RadixSelect.Portal>
+    <RadixSelect.Content
+      ref={ref}
+      className={twMerge(
+        'relative z-50 min-w-[8rem] overflow-hidden rounded bg-white shadow-card-1',
```

Comment:
+1 i'd remove an assumption about the width from here and just provide the value in the place where this component is used, that way you'll probably be able to avoid using min-w